common/src/main/scala/org/apache/spark/sql/comet/util/Utils.scala

```diff
@@ -124,7 +125,10 @@ object Utils {
       case LongType => new ArrowType.Int(8 * 8, true)
       case FloatType => new ArrowType.FloatingPoint(FloatingPointPrecision.SINGLE)
       case DoubleType => new ArrowType.FloatingPoint(FloatingPointPrecision.DOUBLE)
-      case StringType => ArrowType.Utf8.INSTANCE
+      case _: StringType => ArrowType.Utf8.INSTANCE
+      case dt if isStringCollationType(dt) =>
+        // TODO collation information is lost with this transformation
```

Comment:
should we have a ticket?

Reply:
I removed this comment because Spark also does not track this information, as pointed out in https://github.com/apache/datafusion-comet/issues/2190#issuecomment-3215876465 by @parthchandra